test_portale_alloggi/test_portale_alloggi.py

```
@@ -0,0 +1,381 @@
+                    print("-" * 40)
+                
+            except ET.ParseError as e:
+                print(f"⚠️  Could not parse XML response: {e}")
+            
+            print("\n✅ Test request completed successfully!")
+            return True
+            
+        except requests.exceptions.RequestException as e:
+            print(f"❌ HTTP request failed: {e}")
+            return False
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)